browser/app/js/components/Browse.js

```diff
@@ -68,7 +68,7 @@ export default class Browse extends React.Component {
           memory: res.MinioMemory,
           platform: res.MinioPlatform,
           runtime: res.MinioRuntime,
-          envVars: res.MinioEnvVars
+          info: res.MinioGlobalInfo
```

Comment:
You have changes here from master branch.. which is not needed @krisis . Can you clean this up?